genume/constants.py

```diff
@@ -17,3 +24,10 @@
 
 # A list of ignored files when searching for scripts.
 SCRIPTS_IGNORE = [r"__pycache__", r"^\..+", r".*\.md$"]
+
+# Extra executables for bash folder.
+SCRIPTS_BASH_EXTRA = os.path.dirname(os.path.realpath(__file__)) + "/../" + "bash_helpers/"
+
+# Limit the maximum number of child processes to have running at any given time.
+# This limit is imposed by each instance of Registry separately.
+SCRIPTS_MAX_MULTI_DISPATCH = multiprocessing.cpu_count()
```

Comment:
All the scripts are mostly IO operations, so we can afford something like `multiprocessing.cpu_count() * 2`

Reply:
That's not a good idea. You have until tomorrow to figure out why it's a terrible idea and why instead the maximum value should be limited.

Reply:
I did not measure any benefit by having a higher number. It just adds more load to the kernel (as each child creates 3~6 processes).
Also higher multi dispatch numbers may cause longer wait times(when a script is waiting for a reply).